Sources/OrderedCollections/OrderedDictionary/OrderedDictionary.swift

```diff
@@ -272,7 +272,7 @@ extension OrderedDictionary {
   ///     let countryCodes: OrderedDictionary = ["BR": "Brazil", "GH": "Ghana", "JP": "Japan"]
   ///     let index = countryCodes.index(forKey: "JP")
   ///
-  ///     print("Country code for \(countryCodes[index!].value): '\(countryCodes[index!].key)'.")
+  ///     print("Country code for \(countryCodes[offset: index!].value): '\(countryCodes[offset: index!].key)'.")
```

Comment:
Huh, this makes me wonder if got our terminology right.

```swift
let index = countryCodes.index(forKey: "JP")!
print(countryCodes[offset: index].value)
```

Are we calling it an index, or an offset? 🤔